### PR TITLE
Fix live test resources for Azure Monitor

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AppConfigurationExporterIntegrationTest.java [hunk 1]
```diff
@@ -11,78 +15,52 @@
...
             span.end();
             scope.close();
         }
-        assertTrue(appConfigCountDown.await(1, TimeUnit.SECONDS));
-        assertTrue(exporterCountDown.await(1, TimeUnit.SECONDS));
+        assertTrue(appConfigCountDown.await(60, TimeUnit.SECONDS));
+        assertTrue(exporterCountDown.await(60, TimeUnit.SECONDS));
     }
 
+    @Disabled
```
Comment (kryalama) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AppConfigurationExporterIntegrationTest.java:53]: is this test disabled intentional?

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporterTest.java [hunk 2]
```diff
@@ -38,10 +40,14 @@
...
     public void testExportRequestData() {
+        String connectionStringTemplate = "InstrumentationKey=ikey;IngestionEndpoint=https://testendpoint.com";
+        String connectionString = Configuration.getGlobalConfiguration()
+            .get("APPLICATIONINSIGHTS_CONNECTION_STRING", connectionStringTemplate);
         AzureMonitorTraceExporter azureMonitorTraceExporter = getClientBuilder()
-            .connectionString("InstrumentationKey=ikey;IngestionEndpoint=https://testendpoint.com")
+            .connectionString(connectionString)
             .buildTraceExporter();
         CompletableResultCode export = azureMonitorTraceExporter.export(Collections.singleton(new RequestSpanData()));
+        export.join(30, TimeUnit.SECONDS);
```
Comment (kryalama) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporterTest.java:50]: Is this added during "RECORD". can we remove this?

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/EventHubsExporterIntegrationTest.java [hunk 3]
```diff
@@ -69,6 +68,7 @@ public void producerTest() throws InterruptedException {
         assertTrue(exporterCountDown.await(5, TimeUnit.SECONDS));
     }
 
+    @Disabled
```
Comment (kryalama) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/EventHubsExporterIntegrationTest.java:71]: is this disabled intentionally?

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporterTest.java (hunk 2 above)
Comment (srnagar) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporterTest.java:50]: This is a timeout value. So, this is the max it waits but in PLAYBACK mode, it doesn't have to wait for the response and completes immediately.

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/EventHubsExporterIntegrationTest.java (hunk 3 above)
Comment (srnagar) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/EventHubsExporterIntegrationTest.java]: Yes, processor tests run into an issue in CI due to partition contention. This test needs to be updated to use a unique consumer group each time it runs which is currently not configurable through ARM template. This needs more investigation and I have linked an issue to track this.

File: sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml [hunk 4]
```diff
@@ -30,6 +30,11 @@
     <url>https://github.com/Azure/azure-sdk-for-java</url>
   </scm>
 
+  <properties>
+    <jacoco.min.linecoverage>0.30</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0.20</jacoco.min.branchcoverage>
```
[the remark this answers is not in the document]
Comment (srnagar) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml:35]: No, these are current baseline numbers. The coverage should increase as we add more tests and it is not an industry standard.

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AppConfigurationExporterIntegrationTest.java (hunk 1 above)
Comment (srnagar) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AppConfigurationExporterIntegrationTest.java]: Yes, I have an issue to track this and re-enable it.

**Review by kryalama — APPROVED** [review]

**Pull request merged**